synth.py

```diff
@@ -22,12 +22,12 @@
 
 # run the gapic generator
 gapic = gcp.GAPICBazel()
-versions = ['v1beta']
+versions = ['v1', 'v1beta']
 for version in versions:
    library = gapic.node_library('gaming', version)
 s.copy(library, excludes=['package.json', 'README.md'])
 # Copy common templates
 common_templates = gcp.CommonTemplates()
-templates = common_templates.node_library(source_location='build/src')
+templates = common_templates.node_library(source_location='build/src', versions=versions, default_version='v1')
```

Comment:
If this flips the default version, would it be semver major?  Not sure what the changes between `v1beta1` and `v1` look like really

Reply:
I talked to the folks on the team and there should be no breaking changes.